### PR TITLE
update headers and copyrights to correct wording

### PR DESCRIPTION
**Pull request by myronkscott**

### CODE REVIEW

**Comment by cljohnso** [comment]
A question for @chrisdennis and @jhouserizer ... since this project is a test implementation of components that are TPL licensed, is it appropriate to make this Apache License v2?

**Comment by chrisdennis** [comment]
> A question for @chrisdennis and @jhouserizer ... since this project is a test implementation of components that are TPL licensed, is it appropriate to make this Apache License v2?

Yes... TPL is not copy-left it's really just APL with attribution. Totally fine for this to be APL I believe.

File: entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java [hunk 1]
```diff
@@ -157,11 +155,11 @@ private void checkEndpointOpen() {
   }
 
   public long getCurrentId() {
-    return idGenerator.get();
+    return 0L;
```
Comment (cljohnso) [inline entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java:158]: Isn't this a potential behavior change?

File: entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java [hunk 2]
```diff
@@ -157,11 +155,11 @@ private void checkEndpointOpen() {
...
   }
 
   public long resetEldestId() {
-    eldest = idGenerator.get();
+    eldest = 0L;
```
Comment (cljohnso) [inline entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java:162]: Isn't this a potential behavior change?

**Review by cljohnso — COMMENTED** [review]
A question on a couple of lines... otherwise looks good.

File: entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java (hunk 1 above)
Comment (myronkscott) [reply entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java:158]: No.  nothing ever changed it.  constructor value was zero

File: entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java (hunk 2 above)
Comment (myronkscott) [reply entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java:162]: see above

**Review by cljohnso — APPROVED** [review]

**Pull request merged**